sys/vane/dill.hoon

```diff
@@ -459,9 +451,18 @@
                 ?=(^ j.hef.all)
             ==
           +>.$
-        %+  done(hef.all [~ ~ ~ ~ ~ ~ ~])
+        ::
+        =/  ven  hef.all
+        =>  .(hef.all [~ ~ ~ ~ ~ ~ ~])
+        ::
+        %+  done
           %mass
-        =>  [hef.all d=wegh]
+        =/  d=mass  :+  %dill  %|
```

Comment:
Could you add a comment here mentioning that `d` must be computed after `hef.all` has been cleared? Also, I think the noun would be clearer on a newline.

Reply:
done